openedx/core/djangoapps/theming/helpers.py

```diff
@@ -24,7 +32,10 @@ def get_template_path(relative_path, **kwargs):
     """
     This is a proxy function to hide microsite_configuration behind comprehensive theming.
     """
-    return microsite.get_template_path(relative_path, **kwargs)
+    template_path = microsite.get_template_path(relative_path, **kwargs)
+    if not template_path:
+        template_path = get_template_path_with_theme(relative_path)
```

Comment:
I feel like the majority case in the near future will be theme vs. microsite, so I am wondering if this code should check for theme in the first place, then if no theme found we check for microsite -- thoughts?